[2023-10-15 10:05:13,060: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 10:05:13,103: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:05:13,145: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:05:13,153: INFO: common: created directory at: artifacts]
[2023-10-15 10:05:13,170: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 10:05:19,540: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9982:15F91D:44413:82C62:652B6C00
Accept-Ranges: bytes
Date: Sun, 15 Oct 2023 04:35:15 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100021-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1697344514.170096,VS0,VE922
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: ad25a8b88f0184e9c5ccabef01c6c490d23a06ff
Expires: Sun, 15 Oct 2023 04:40:15 GMT
Source-Age: 1

]
[2023-10-15 10:05:21,662: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-10-15 10:05:21,662: INFO: main: *******************]
[2023-10-15 10:05:21,662: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2023-10-15 10:05:21,670: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:05:21,695: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:05:21,721: INFO: common: created directory at: artifacts]
[2023-10-15 10:05:21,747: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 10:05:25,144: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 10:05:28,029: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2023-10-15 10:05:28,029: INFO: main: *******************]
[2023-10-15 10:05:28,029: INFO: main: >>>>>> stage Training started <<<<<<]
[2023-10-15 10:05:28,029: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:05:28,045: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:05:28,045: INFO: common: created directory at: artifacts]
[2023-10-15 10:05:28,061: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 10:05:28,070: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 10:05:28,075: ERROR: main: 'WindowsPath' object has no attribute 'endswith']
Traceback (most recent call last):
  File "C:\ML Projects\Chicken-Disease-Classification-Projects\main.py", line 40, in <module>
    model_trainer.main()
  File "c:\ml projects\chicken-disease-classification-projects\src\cnnClassifier\pipeline\stage_03_training.py", line 19, in main
    callback_list = prepare_callbacks.get_tb_ckpt_callbacks()
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "c:\ml projects\chicken-disease-classification-projects\src\cnnClassifier\components\prepare_callbacks.py", line 36, in get_tb_ckpt_callbacks
    self._create_ckpt_callbacks
  File "c:\ml projects\chicken-disease-classification-projects\src\cnnClassifier\components\prepare_callbacks.py", line 27, in _create_ckpt_callbacks
    return tf.keras.callbacks.ModelCheckpoint(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\site-packages\keras\src\callbacks.py", line 1369, in __init__
    if filepath and filepath.endswith(".keras") and options is not None:
                    ^^^^^^^^^^^^^^^^^
AttributeError: 'WindowsPath' object has no attribute 'endswith'
[2023-10-15 10:34:10,927: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:34:10,967: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:34:10,975: INFO: common: created directory at: artifacts]
[2023-10-15 10:34:10,983: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 10:34:10,991: INFO: 1170291011: File already exists of size: ~ 11345 KB]
[2023-10-15 10:36:41,911: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:36:41,927: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:36:41,927: INFO: common: created directory at: artifacts]
[2023-10-15 10:36:41,935: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 10:36:45,804: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 10:38:03,495: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:38:03,511: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:38:03,519: INFO: common: created directory at: artifacts]
[2023-10-15 10:38:03,527: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 10:38:03,535: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 10:47:11,541: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:47:11,549: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:47:11,555: INFO: common: created directory at: artifacts]
[2023-10-15 10:47:11,555: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 10:47:11,563: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 10:47:30,538: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:47:30,546: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:47:30,546: INFO: common: created directory at: artifacts]
[2023-10-15 10:47:30,554: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 10:47:30,562: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 10:53:31,164: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 10:53:31,180: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:53:31,188: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:53:31,188: INFO: common: created directory at: artifacts]
[2023-10-15 10:53:31,188: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 10:53:31,196: INFO: data_ingestion: File already exists of size: ~ 11345 KB]
[2023-10-15 10:53:34,087: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-10-15 10:53:34,087: INFO: main: *******************]
[2023-10-15 10:53:34,087: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2023-10-15 10:53:34,102: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:53:34,102: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:53:34,102: INFO: common: created directory at: artifacts]
[2023-10-15 10:53:34,102: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 10:53:37,183: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 10:53:40,648: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2023-10-15 10:53:40,648: INFO: main: *******************]
[2023-10-15 10:53:40,648: INFO: main: >>>>>> stage Training started <<<<<<]
[2023-10-15 10:53:40,648: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 10:53:40,666: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 10:53:40,666: INFO: common: created directory at: artifacts]
[2023-10-15 10:53:40,666: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 10:53:40,674: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 10:53:40,682: INFO: common: created directory at: artifacts\training]
[2023-10-15 11:01:49,144: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2023-10-15 11:01:49,160: INFO: main: *******************]
[2023-10-15 11:01:49,160: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2023-10-15 11:01:49,277: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 11:01:49,297: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 11:01:49,313: INFO: common: created directory at: artifacts]
[2023-10-15 11:04:25,443: INFO: common: json file saved at: scores.json]
[2023-10-15 11:04:25,443: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2023-10-15 11:17:52,249: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:80
 * Running on http://192.168.0.121:80]
[2023-10-15 11:17:52,257: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-10-15 11:18:19,874: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 11:18:19] "GET / HTTP/1.1" 200 -]
[2023-10-15 11:20:09,375: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 11:20:09] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 11:22:18,241: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 11:22:18,259: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 11:22:18,278: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 11:22:18,278: INFO: common: created directory at: artifacts]
[2023-10-15 11:22:18,286: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 11:22:19,711: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Content-Type: text/html; charset=utf-8
X-Robots-Tag: noindex, nofollow, nosnippet
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Pragma: no-cache
Expires: Mon, 01 Jan 1990 00:00:00 GMT
Date: Sun, 15 Oct 2023 05:52:19 GMT
P3P: CP="This is not a P3P policy! See g.co/p3phelp for more info."
Content-Security-Policy: require-trusted-types-for 'script';report-uri https://csp.withgoogle.com/csp/docs-tt
Referrer-Policy: origin
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
Server: GSE
Set-Cookie: NID=511=BrPqwRG5xX1CfOp-twLYE3QfT7pFfaFoSXmlrvlehNN_DRvGrM0duQtTPfxM6oso10Vi2IZxEq5RR4nvz5Jp3oXveNKUbuO4CrXNKOvJ2mM09MlQp6U70vFlKcB5BubroErmTLbRW4VUPBpxYyY41TY2hcQTwz-JynHfLJym6ro; expires=Mon, 15-Apr-2024 05:52:19 GMT; path=/; domain=.google.com; HttpOnly
Alt-Svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
Accept-Ranges: none
Vary: Sec-Fetch-Dest, Sec-Fetch-Mode, Sec-Fetch-Site,Accept-Encoding
Connection: close
Transfer-Encoding: chunked

]
[2023-10-15 11:22:20,358: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "C:\ML Projects\Chicken-Disease-Classification-Projects\main.py", line 12, in <module>
    data_ingestion.main()
  File "c:\ml projects\chicken-disease-classification-projects\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 17, in main
    data_ingestion.extract_zip_file()
  File "c:\ml projects\chicken-disease-classification-projects\src\cnnClassifier\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1299, in __init__
    self._RealGetContents()
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1366, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-10-15 11:41:17,687: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 11:41:17,703: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 11:41:17,735: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 11:41:17,743: INFO: common: created directory at: artifacts]
[2023-10-15 11:41:17,751: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 11:41:17,759: INFO: data_ingestion: File already exists of size: ~ 82 KB]
[2023-10-15 11:41:17,782: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\main.py", line 12, in <module>
    data_ingestion.main()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 17, in main
    data_ingestion.extract_zip_file()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1299, in __init__
    self._RealGetContents()
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1366, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-10-15 11:43:38,108: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 11:43:38,117: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 11:43:38,124: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 11:43:38,124: INFO: common: created directory at: artifacts]
[2023-10-15 11:43:38,134: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 11:43:38,136: INFO: data_ingestion: File already exists of size: ~ 82 KB]
[2023-10-15 11:43:38,144: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\main.py", line 12, in <module>
    data_ingestion.main()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 17, in main
    data_ingestion.extract_zip_file()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1299, in __init__
    self._RealGetContents()
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1366, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-10-15 11:49:40,525: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 11:49:40,552: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 11:49:40,560: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 11:49:40,568: INFO: common: created directory at: artifacts]
[2023-10-15 11:49:40,568: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 11:49:40,576: INFO: data_ingestion: File already exists of size: ~ 82 KB]
[2023-10-15 11:49:40,593: ERROR: main: File is not a zip file]
Traceback (most recent call last):
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\main.py", line 12, in <module>
    data_ingestion.main()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 17, in main
    data_ingestion.extract_zip_file()
  File "C:\ML Projects\Potato-Leaf-Desease-End2End\src\cnnClassifier\components\data_ingestion.py", line 36, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1299, in __init__
    self._RealGetContents()
  File "C:\Users\ACER\anaconda3\envs\leaf_desease_detection\Lib\zipfile.py", line 1366, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2023-10-15 12:07:51,128: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:07:51,158: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:07:51,158: INFO: common: created directory at: artifacts]
[2023-10-15 12:07:51,166: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 12:07:57,802: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 12:09:11,408: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:09:11,424: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:09:11,424: INFO: common: created directory at: artifacts]
[2023-10-15 12:09:11,432: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:09:11,443: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:11:08,243: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:11:08,259: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:11:08,266: INFO: common: created directory at: artifacts]
[2023-10-15 12:11:08,274: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:11:08,274: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:13:17,395: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:13:17,403: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:13:17,411: INFO: common: created directory at: artifacts]
[2023-10-15 12:13:17,411: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:13:17,422: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:13:17,430: INFO: common: created directory at: artifacts\training]
[2023-10-15 12:13:23,582: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:13:23,590: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:13:23,590: INFO: common: created directory at: artifacts]
[2023-10-15 12:13:23,598: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:13:23,606: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:13:23,606: INFO: common: created directory at: artifacts\training]
[2023-10-15 12:15:08,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:15:08,578: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:15:08,586: INFO: common: created directory at: artifacts]
[2023-10-15 12:15:08,594: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:15:08,594: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:15:08,602: INFO: common: created directory at: artifacts\training]
[2023-10-15 12:23:02,402: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:23:02,427: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:23:02,456: INFO: common: created directory at: artifacts]
[2023-10-15 12:23:02,470: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:23:02,476: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:23:02,567: INFO: common: created directory at: artifacts\training]
[2023-10-15 12:26:35,563: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:26:35,574: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:26:35,580: INFO: common: created directory at: artifacts]
[2023-10-15 12:26:35,586: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 12:26:46,553: INFO: 1170291011: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 21137678
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "3018953fda39d25f745a547addfd47d653799e3f5dfaaeca8a1c3e778fc37d06"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7D5A:D8C7B:BEE31:14EB29:652B8D23
Accept-Ranges: bytes
Date: Sun, 15 Oct 2023 06:56:38 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100031-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1697352997.826741,VS0,VE1281
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: be32b5535950a26695644201432d6d72781223d4
Expires: Sun, 15 Oct 2023 07:01:38 GMT
Source-Age: 1

]
[2023-10-15 12:28:19,295: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:28:19,304: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:28:19,309: INFO: common: created directory at: artifacts]
[2023-10-15 12:28:19,319: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 12:28:24,066: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:28:24,081: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:28:24,086: INFO: common: created directory at: artifacts]
[2023-10-15 12:28:24,099: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:28:24,106: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:28:24,845: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 12:29:22,900: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:29:22,930: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:29:22,944: INFO: common: created directory at: artifacts]
[2023-10-15 12:29:22,959: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 12:29:22,966: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 12:29:23,087: INFO: common: created directory at: artifacts\training]
[2023-10-15 12:58:58,729: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 12:58:58,741: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 12:58:58,746: INFO: common: created directory at: artifacts]
[2023-10-15 13:00:00,638: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 13:00:00,647: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 13:00:00,655: INFO: common: created directory at: artifacts]
[2023-10-15 13:09:01,575: INFO: common: json file saved at: scores.json]
[2023-10-15 13:10:46,979: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2023-10-15 13:10:46,985: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 13:10:46,988: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 13:10:46,993: INFO: common: created directory at: artifacts]
[2023-10-15 13:10:46,993: INFO: common: created directory at: artifacts/data_ingestion]
[2023-10-15 13:10:46,998: INFO: data_ingestion: File already exists of size: ~ 20642 KB]
[2023-10-15 13:10:53,253: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2023-10-15 13:10:53,263: INFO: main: *******************]
[2023-10-15 13:10:53,308: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2023-10-15 13:10:53,344: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 13:10:53,394: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 13:10:53,414: INFO: common: created directory at: artifacts]
[2023-10-15 13:10:53,444: INFO: common: created directory at: artifacts/prepare_base_model]
[2023-10-15 13:10:56,919: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2023-10-15 13:11:00,186: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2023-10-15 13:11:00,191: INFO: main: *******************]
[2023-10-15 13:11:00,196: INFO: main: >>>>>> stage Training started <<<<<<]
[2023-10-15 13:11:00,231: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 13:11:00,246: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 13:11:00,251: INFO: common: created directory at: artifacts]
[2023-10-15 13:11:00,251: INFO: common: created directory at: artifacts\prepare_callbacks\checkpoint_dir]
[2023-10-15 13:11:00,261: INFO: common: created directory at: artifacts\prepare_callbacks\tensorboard_log_dir]
[2023-10-15 13:11:00,301: INFO: common: created directory at: artifacts\training]
[2023-10-15 13:35:41,306: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2023-10-15 13:35:41,341: INFO: main: *******************]
[2023-10-15 13:35:41,341: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2023-10-15 13:35:41,611: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-10-15 13:35:41,637: INFO: common: yaml file: params.yaml loaded successfully]
[2023-10-15 13:35:41,647: INFO: common: created directory at: artifacts]
[2023-10-15 13:42:29,969: INFO: common: json file saved at: scores.json]
[2023-10-15 13:42:29,972: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2023-10-15 13:43:41,402: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:80
 * Running on http://192.168.0.121:80]
[2023-10-15 13:43:41,412: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-10-15 13:43:49,960: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:43:49] "GET / HTTP/1.1" 200 -]
[2023-10-15 13:45:14,779: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:45:14] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:46:43,045: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:46:43] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:46:54,773: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:46:54] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:47:39,370: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:47:39] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:47:55,724: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:47:55] "GET / HTTP/1.1" 200 -]
[2023-10-15 13:48:07,990: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000002B0F8E68EA0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2023-10-15 13:48:08,002: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:48:08] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:48:19,442: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000002B0803B1800> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2023-10-15 13:48:19,457: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:48:19] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:50:45,841: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:50:45] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:52:52,753: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:52:52] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:53:11,422: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:53:11] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:54:25,291: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:54:25] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:55:07,016: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:80
 * Running on http://192.168.0.121:80]
[2023-10-15 13:55:07,021: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2023-10-15 13:55:13,918: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:55:13] "GET / HTTP/1.1" 200 -]
[2023-10-15 13:55:29,326: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:55:29] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:55:45,417: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:55:45] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:55:59,843: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:55:59] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:56:17,041: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:56:17] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:57:01,159: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:57:01] "GET / HTTP/1.1" 200 -]
[2023-10-15 13:57:13,791: WARNING: polymorphic_function: 5 out of the last 5 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000001FAC6778F40> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2023-10-15 13:57:13,806: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:57:13] "POST /predict HTTP/1.1" 200 -]
[2023-10-15 13:57:29,166: WARNING: polymorphic_function: 6 out of the last 6 calls to <function Model.make_predict_function.<locals>.predict_function at 0x000001FAC80B18A0> triggered tf.function retracing. Tracing is expensive and the excessive number of tracings could be due to (1) creating @tf.function repeatedly in a loop, (2) passing tensors with different shapes, (3) passing Python objects instead of tensors. For (1), please define your @tf.function outside of the loop. For (2), @tf.function has reduce_retracing=True option that can avoid unnecessary retracing. For (3), please refer to https://www.tensorflow.org/guide/function#controlling_retracing and https://www.tensorflow.org/api_docs/python/tf/function for  more details.]
[2023-10-15 13:57:29,181: INFO: _internal: 192.168.0.121 - - [15/Oct/2023 13:57:29] "POST /predict HTTP/1.1" 200 -]
